test/integration/etcd_storage_path_test.go

```diff
@@ -485,19 +495,10 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 		stub:             `{"metadata": {"name": "deployment1"}, "spec": {"selector": {"matchLabels": {"f": "z"}}, "template": {"metadata": {"labels": {"f": "z"}}, "spec": {"containers": [{"image": "fedora:latest", "name": "container6"}]}}}}`,
 		expectedEtcdPath: "kubernetes.io/deployments/etcdstoragepathtestnamespace/deployment1",
 	},
-	gvr("extensions", "v1beta1", "horizontalpodautoscalers"): {
```

Comment:
So that kubecompat is only for reading/storing them as extensions/v1beta1 in etcd?